### PR TITLE
[R20-295] Adding error pages

### PR DESCRIPTION
**Pull request by lambry**
**Issue**: https://digital-vic.atlassian.net/browse/R20-295

#### Notes
- Using default themed pattern for corner graphic, may come from global CMS option later
- Ripple 1 has the contact link on the error page hard coded, this is the same here for now i.e. `/connect-with-us`, this seems to always redirect to the correct contact page URL.

### What I did

- Added base error message component
- Added Error page for 404/500 errors

### How to test

- The component is viewable in storybook
- Cypress tests
- Reference site? 

### Screenshot
400-error
500-error

#### For all PR's

- [ ] I've added relevant changes to the project Readme if needed.
- [ ] I've updated the documentation site as needed.
- [ ] I have added unit tests to cover my changes (if not applicable, please state why in a comment)

#### For new components only

- [ ] I have added a story covering all variants
- [x] I have checked a11y tab in storybook passes
- [ ] Any events are emitted on the event bus

### CODE REVIEW

**Comment by dylankelly** [comment]
@lambry Re: the corner graphics - Ripple 1 implemented custom header graphics per site by placing the graphic in the FE repo - Eg https://github.com/dpc-sdp/ripple/blob/develop/examples/basic-examples/static/img/header-pattern-shape.png, https://github.com/dpc-sdp/vicpol-vic-gov-au/blob/develop/static/img/header-pattern-shape.png

I would like to move away from this in favour of having these come from the CMS in Ripple 2 - I'll see if we can add a ticket for the BE to consider adding it to the site taxonomy

File: packages/ripple-tide-api/src/nuxt/components/ContentPage.vue [hunk 1]
```diff
@@ -58,6 +48,15 @@ const { data: site, error: siteError } = await useFetch('/api/tide/site', {
     id: siteId
   }
 })
+
+if (siteError.value) {
```
Comment (dylankelly) [inline packages/ripple-tide-api/src/nuxt/components/ContentPage.vue:52]: @lambry - I'm wondering if these errors would be better handled in a composable? Would be good to keep the setup function cleaner and also the error logic would potentially be able to be reused in pages that don't implement `ContentPage` component

File: packages/ripple-tide-api/src/nuxt/components/ErrorPage.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <template #breadcrumbs><span></span></template>
+    <template #aboveBody>
+      <RplHeaderGraphic :image="true" placement="top" />
+    </template>
+    <template #body>
+      <RplErrorMessage
+        :title="title"
+        :intro="props.error?.statusMessage"
+        :link="{ url: '/', text: 'Go back home' }"
+        :data-cy="`error-${props.error.statusCode}`"
```
Comment (dylankelly) [inline packages/ripple-tide-api/src/nuxt/components/ErrorPage.vue:16]: I think we need to discuss how we want to write Cypress tests - adding these `data-cy` attributes everywhere could get a bit messing and adds page weight to prod sites. CC @waitingallday

**Review by dylankelly — APPROVED** [review]
@lambry - nice one - few things here as discussion points.

File: packages/ripple-tide-api/src/nuxt/components/ErrorPage.vue (hunk 2 above)
Comment (waitingallday) [reply packages/ripple-tide-api/src/nuxt/components/ErrorPage.vue:16]: Let's get rid of them before we go to prod

**Review by waitingallday — APPROVED** [review]

**Pull request merged**